compiler/src/testLite/golden/TestDeprecatedService.java.txt

```diff
@@ -7,9 +7,6 @@ import static io.grpc.MethodDescriptor.generateFullMethodName;
  * Test service that has been deprecated and should generate with Java's &#64;Deprecated annotation
  * </pre>
  */
-@javax.annotation.Generated(
```

Comment:
@ejona86 - do you think we need another variant of this test to cover the case where `@javax.annotation.Generated` is not skipped?

Reply:
That's kept in `test/golden`
https://github.com/grpc/grpc-java/blob/add8c37a4148f6dd857dd31c7425e2ae526e0528/compiler/src/test/golden/TestDeprecatedService.java.txt#L10-L15

Reply:
Yeah, we're "abusing" the fact we have lite vs normal and only doing this is the lite version.

Reply:
Ah, I see.